client/css/style.css

```diff
@@ -1260,6 +1260,7 @@ button {
 }
 
 #form #input {
+	overflow-y: hidden;
```

Comment:
This will break scrolling when it hits 6-7 lines, as we stop growing at this point.